autogpt_platform/backend/backend/api/features/experts/scheduling.py

```
@@ -0,0 +1,303 @@
+        )
+    except Exception as e:
+        logger.warning(
+            f"Schedule for expert #{expert_id} workflow #{workflow_row_id} "
+            f"not created (needs setup): {type(e).__name__}: {e}"
+        )
+        return False
+    try:
+        await prisma.models.ExpertWorkflow.prisma().update(
+            where={"id": workflow_row_id}, data={"scheduleId": schedule.id}
+        )
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)